src/stars.css

```diff
@@ -0,0 +1,4 @@
+.stars {
+  display: flex;
+  justify-content: center;
```

Comment:
Not a mistake, just a reminder, you don't need to use this justify-content on this project. It's not wrong, you simply don't need it.